sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -214,14 +214,18 @@ final class TypeSig private(
 
   /**
    * Given an expression tag the associated meta for it to be supported or not.
-   * @param meta the meta that gets marked for support or not.
-   * @param expr the expression to check against.
-   * @param name the name of the expression (typically a parameter name)
+   *
+   * @param meta     the meta that gets marked for support or not.
+   * @param exprMeta the meta of expression to check against.
+   * @param name     the name of the expression (typically a parameter name)
    */
-  def tagExprParam(meta: RapidsMeta[_, _, _], expr: Expression, name: String): Unit = {
+  def tagExprParam(meta: RapidsMeta[_, _, _],
+      exprMeta: BaseExprMeta[_],
```

Comment:
nit: Why split this up onto multiple lines? it is not any longer than it was before?

Reply:
I reverted this change.